### PR TITLE
GRVDEV-167 Add Ability to Inject WAF logs

### PR DESCRIPTION
**Pull request by TysonBeck**

### CODE REVIEW

File: aws/arctic-wolf/arctic_wolf.tf [hunk 1]
```diff
@@ -55,6 +55,30 @@ variable "cloudtrail_name" {
   description = ""
 }
 
+variable "waf_arn" {
+  type        = map
+  default     = null
+  description = ""
+}
+
+variable "log_bucket_id" {
```
Comment (jeremyodle) [inline aws/arctic-wolf/arctic_wolf.tf]: change to `cloudtrail_bucket_id` if logging is specific to cloudtrail

File: aws/arctic-wolf/arctic_wolf.tf [hunk 2]
```diff
@@ -197,6 +225,277 @@ resource "aws_cloudformation_stack" "vpc_flow_log_group" {
...
+
+  server_side_encryption {
+    enabled = true
+  }
+
+  extended_s3_configuration {
+    role_arn           = aws_iam_role.kinesis[0].arn
+    bucket_arn         = var.cloudtrail_name != null ? var.log_bucket_arn : aws_s3_bucket.default[0].arn
+    s3_backup_mode     = "Disabled"
+    prefix             = "AWN/WAF/${var.account_id}/"
```
Comment (jeremyodle) [inline aws/arctic-wolf/arctic_wolf.tf]: Refactor `AWN/WAF/` to a local variable

File: aws/arctic-wolf/arctic_wolf.tf [hunk 3]
```diff
@@ -197,6 +225,277 @@ resource "aws_cloudformation_stack" "vpc_flow_log_group" {
...
+  topic {
+    topic_arn     = "arn:aws:sns:${var.aws_region}:${var.account_id}:AWNSNSTopic"
+    events        = ["s3:ObjectCreated:*"]
+    filter_prefix = "AWN/WAF/${var.account_id}/"
+  }
+}
+
 # ----------------------------------------------------------------------------------------------------------------------
 # Outputs
 # ----------------------------------------------------------------------------------------------------------------------
```
Comment (jeremyodle) [inline aws/arctic-wolf/arctic_wolf.tf:538]: Add any relative outputs

**Review by jeremyodle — CHANGES_REQUESTED** [review]

File: aws/arctic-wolf/arctic_wolf.tf (hunk 1 above)
Comment (TysonBeck) [reply aws/arctic-wolf/arctic_wolf.tf]: Updated variable naming

File: aws/arctic-wolf/arctic_wolf.tf (hunk 2 above)
Comment (TysonBeck) [reply aws/arctic-wolf/arctic_wolf.tf]: Switched prefix to a local variable

File: aws/arctic-wolf/arctic_wolf.tf (hunk 3 above)
Comment (TysonBeck) [reply aws/arctic-wolf/arctic_wolf.tf:505]: Updated outputs

**Review by jeremyodle — DISMISSED** [review]

File: aws/arctic-wolf/arctic_wolf.tf (hunk 3 above)
Comment (TysonBeck) [reply aws/arctic-wolf/arctic_wolf.tf:538]: Added outputs

File: aws/arctic-wolf/arctic_wolf.tf [hunk 4]
```diff
@@ -2,83 +2,89 @@
...
 }
 
-variable "cloudtrail_name" {
+variable cloudtrail_name {
   type        = string
   default     = null
   description = ""
 }
 
 variable "waf_arn" {
```
Comment (jeremyodle) [inline aws/arctic-wolf/arctic_wolf.tf]: - [x] `waf_arns`

**Review by jeremyodle — CHANGES_REQUESTED** [review]
- [x] Move `tags = local.tags` to the top of the resource

File: aws/arctic-wolf/arctic_wolf.tf (hunk 4 above)
Comment (TysonBeck) [reply aws/arctic-wolf/arctic_wolf.tf]: updated to waf_arns

**Review by jeremyodle — APPROVED** [review]

**Pull request merged**